clients/web/src/views/widgets/FolderInfoWidget.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * This view allows users to see and control access on a resource.
+ */
+girder.views.FolderInfoWidget = girder.View.extend({
+    initialize: function () {
+        this.needToFetch = !this.model.has('nItems') || !this.model.has('nFolders');
+        if (this.needToFetch) {
+            this.model.fetch({extraPath: 'details'}).once('g:fetched.details', function () {
```

Comment:
Good find. It's possible, but I'd have to probably write a separate unit test for just this widget, which might not be a bad idea. (Right now the details get fetched and cached as soon as the folder is loaded as the current hierarchy widget view.)